recOrder/acq/acquisition_workers.py

```diff
@@ -1236,14 +1235,14 @@ def _cleanup_acq(self):
                         else f"{save_prefix}_RawPolDataSnap.zarr"
                     )
                     out_path = os.path.join(self.snap_dir, name)
-                    converter = ZarrConverter(
+                    converter = TIFFConverter(
                         os.path.join(dir_, prefix),
                         out_path,
-                        "ometiff",
-                        False,
-                        False,
+                        data_type="ometiff",
+                        grid_layout=False,
+                        label_positions=False,
                     )
-                    converter.run_conversion()
+                    converter.run()
```

Comment:
@ziw-liu I'm getting the following traceback when I try to run the converter from within `recOrder`:

Related to #266? I'll test the iohub converter on windows.

```
File ~\recOrder\recOrder\acq\acquisition_workers.py:1245, in PolarizationAcquisitionWorker._cleanup_acq(self=<recOrder.acq.acquisition_workers.PolarizationAcquisitionWorker object>)
   1237     out_path = os.path.join(self.snap_dir, name)
   1238     converter = TIFFConverter(
   1239         os.path.join(dir_, prefix),
   1240         out_path,
   (...)
   1243         label_positions=False,
   1244     )
-> 1245     converter.run()
        converter = <iohub.convert.TIFFConverter object at 0x0000022A812E3EE0>
   1246     shutil.rmtree(os.path.join(dir_, prefix))
   1247 except PermissionError as ex:

File ~\.conda\envs\recorder-dev\lib\site-packages\iohub\convert.py:331, in TIFFConverter.run(self=<iohub.convert.TIFFConverter object>, check_image=True)
    322 """Runs the conversion.
    323 
    324 Parameters
   (...)
    328     pixel values as in TIFF files, by default True
    329 """
    330 logging.debug("Setting up Zarr store.")
--> 331 self._init_hcs_arrays()
        self = <iohub.convert.TIFFConverter object at 0x0000022A812E3EE0>
    332 bar_format = (
    333     "Status: |{bar:16}|{n_fmt}/{total_fmt} "
    334     "(Time Remaining: {remaining}), {rate_fmt}{postfix}]"
    335 )
    336 # Run through every coordinate and convert in acquisition order

File ~\.conda\envs\recorder-dev\lib\site-packages\iohub\convert.py:304, in TIFFConverter._init_hcs_arrays(self=<iohub.convert.TIFFConverter object>)
    302 for column in columns:
    303     pos_name = self.pos_names[len(self.well_list)]
--> 304     pos = self.writer.create_position(row, column, pos_name="0")
        self.writer = <iohub.ngff.Plate object at 0x0000022A812FB850>
        self = <iohub.convert.TIFFConverter object at 0x0000022A812E3EE0>
        row = 0
        column = 0
    305     self.well_list.append(os.path.join(str(row), str(column)))
    306     _ = pos.zgroup.zeros(
    307         "0",
    308         shape=(
   (...)
    315         chunks=self.chunks,
    316     )

File ~\.conda\envs\recorder-dev\lib\site-packages\iohub\ngff.py:1331, in Plate.create_position(self=<iohub.ngff.Plate object>, row_name='0', col_name='0', pos_name='0', row_index=None, col_index=None, acq_index=0)
   1329     well = self[well_path]
   1330 else:
-> 1331     well = self.create_well(
        self = <iohub.ngff.Plate object at 0x0000022A812FB850>
        row_name = '0'
        col_name = '0'
        row_index = None
        col_index = None
   1332         row_name, col_name, row_index=row_index, col_index=col_index
   1333     )
   1334 return well.create_position(pos_name, acquisition=acq_index)

File ~\.conda\envs\recorder-dev\lib\site-packages\iohub\ngff.py:1261, in Plate.create_well(self=<iohub.ngff.Plate object>, row_name='0', col_name='0', row_index=0, col_index=0)
   1259 col_index = self._auto_idx(col_index, self._cols)
   1260 # build well metadata
-> 1261 well_index_meta = WellIndexMeta(
        WellIndexMeta = <class 'iohub.ngff_meta.WellIndexMeta'>
        row_name = '0'
        col_name = '0'
        os.path = <module 'ntpath' from 'C:\\Users\\LabelFree\\.conda\\envs\\recorder-dev\\lib\\ntpath.py'>
        row_index = 0
        col_index = 0
   1262     path=os.path.join(row_name, col_name),
   1263     row_index=row_index,
   1264     column_index=col_index,
   1265 )
   1266 col_meta = PlateAxisMeta(name=col_name)
   1267 # create new row if needed

File ~\.conda\envs\recorder-dev\lib\site-packages\pydantic\main.py:341, in pydantic.main.BaseModel.__init__()

ValidationError: 1 validation error for WellIndexMeta
path
  The well path '0\0' is not in the form of 'row/column'! (type=value_error)
```

Reply:
I just confirmed that I get the same error if I try to convert an MM dataset using `iohub convert` on the hummingbird machine. 

@ziw-liu Do you think there's a way to fix this issue on the iohub side? Or does it require a fix on the hummingbird computer?

Reply:
Resolved with [iohub #102](https://github.com/czbiohub/iohub/pull/102). 

I'll continue testing on this branch with that commit, and I'll update the `iohub` tags when this is ready to review.